Makefile

```diff
@@ -13,6 +13,8 @@ ETCDIR ?= ${DESTDIR}/etc
 ETCDIR_OCID ?= ${ETCDIR}/ocid
 BUILDTAGS := selinux seccomp $(shell hack/btrfs_tag.sh) $(shell hack/libdm_tag.sh)
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
+GOBINDIR := $(word 1,$(subst :, ,$(GOPATH)))
+PATH := $(GOBINDIR)/bin:$(PATH)
```

Comment:
@jawnsy We will go with this approach for now and revisit if there are issues later. Thanks!